tests/test_worker_palavras_raw.py

```diff
@@ -28,6 +29,7 @@
 
 class TestPalavrasRawWorker(TaskTest):
 
+    @skipIf(not palavras_raw.palavras_installed(), 'palavras software is not installed')
     def test_should_run_only_if_language_is_portuguese(self):
```

Comment:
I think it makes sense to keep it. Beacuse we have to look if the worker is running anyways. The name of the test can be changed to `test_run_worker_if_language_is_portuguese`. A test that says **run only** should try a different language, which I think is not in the context of this PR.